owtf/db/url_manager.py

```diff
@@ -105,6 +105,19 @@ def add_to_db(self, url, visited, found=None, target_id=None):
             self.db.session.merge(models.Url(target_id=target_id, url=url, visited=visited, scope=scope))
             self.db.session.commit()
 
+    def get_num_urls(self, target_id=None):
+        count = self.db.session.query(models.Url(target_id=target_id)).count()
+        return count
+
+    def add_urls_start(self):
```

Comment:
Is the name of the function accurate? I am not sure to understand its purpose when I read `add_urls_start` and I see that it stores the number of URLs to a class attribute.

Reply:
I ported it from a commit while back because the plugin helper had a method which was calling it. I removed it from there.